mineinabyss-components/src/main/kotlin/com/mineinabyss/components/cosmetics/Cosmetics.kt

```diff
@@ -10,7 +10,8 @@ import org.bukkit.entity.Player
 @Serializable
 @SerialName("mineinabyss:cosmetics")
 class Cosmetics(
-    var gesture: @Contextual Gesture? = null
+    var gesture: @Contextual Gesture? = null,
```

Comment:
Best if this isn't a var, same for all other components and mutable lists if they don't need to be changed often, see https://wiki.mineinabyss.com/geary/guide/designing-components/#aim-for-immutable-components

Reply:
Its unused for now so ill remove it